grammars/jsdoc.cson

```diff
@@ -411,6 +435,120 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Double-quoted string
+      {
+        'begin': '(")'
```

Comment:
Alright, it's done. I've merged both string-matching blocks, and used `string.quoted.js` for the applied scope.

If there's anything else that's blocking this, please speak up.